lua/lvim/core/alpha.lua

```diff
@@ -36,13 +39,16 @@ local function resolve_config(theme_name)
   for name, el in pairs(section) do
     for k, v in pairs(el) do
       if name:match "buttons" and k == "entries" then
-        resolved_section[name].val = resolve_buttons(theme_name, v)
+        resolved_section[name].val = resolve_buttons(theme_name, v, el.opts)
       elseif v then
         resolved_section[name][k] = v
       end
     end
   end
 
+  local opts = lvim.builtin.alpha[theme_name].opts or {}
+  selected_theme.config.opts = vim.tbl_extend("force", selected_theme.config.opts, opts)
+
```

Comment:
```suggestion
```

Reply:
that's needed for `lvim.builtin.alpha.dashboard.opts = {autostart = false}`